packages/app/src/common/utils.ts

```diff
@@ -81,3 +76,18 @@ export function debounce<F extends (...args: Parameters<F>) => ReturnType<F>>(
     timeout = setTimeout(() => func(...args), waitFor);
   };
 }
+
+export { MAX_NUM_PRECISION, toSol } from "@sunrisestake/client";
```

Comment:
nit:

Is the idea here that the rest of the app will not have a dependency on the client but just on this file? My preference would be to leave this line out, and just have any parts of the app that need it just import these directly from the client.

Reply:
Makes sense, will change it that way.